Battlegroup.cat

```diff
@@ -112,7 +112,6 @@
       </categoryLinks>
     </entryLink>
     <entryLink id="warmaster-support-battlegroup" name="Warmaster Titan" hidden="false" collective="false" import="false" targetId="2062-8f97-c49e-abe2" type="selectionEntry"/>
-    <entryLink id="warhound-squadron-support-battlegroup" name="Warhound Squadron" hidden="false" collective="false" import="false" targetId="9fc6-f5ca-16a8-7515" type="selectionEntry"/>
```

Comment:
I can see why this might cause a problem because I cannot find any other elements with an Id which matches this elements targetId.

But for my own education, how did you see this was an error in the Data Editor?

Reply:
the battlescribe data editor itself highlighted the issue when I tried to save my fix to issue #140, I only removed the broken link after failing to find the missing elements target ID in the "Battlegroup.cat" file

Reply:
the error did not appear in the Battlescribe Roster Editor, it only appeared in the Battlescribe Data Editor